### ISSUE TITLE
Generating Unwanted Loot on "Better Table" and Journal Roll Buttons

### ISSUE THREAD

**Issue by profdresser** [ultrakorne/better-rolltables#116]
My understanding was that loot would only be generated from a text entry in a Loot Table. If I'm wrong about that, I'm extremely sorry. 
I'm getting this weird thing where it generates loot and also seems to search for a compendium when I get a text result from a Roll+ roll on a non-Loot Table and also when I use the spiffy new journal roll button. I'm sure it wasn't doing this with the Better Tables before, but that was a while ago.

**Basic Tables**
Basic Table w/ Text Entries
![Screenshot 2021-07-31 105451](https://user-images.githubusercontent.com/88245061/127745593-bb3e7a49-0cb5-4e9d-a462-7ea3a3d81dfe.png)
Roll Result w/ Basic Table
![Screenshot 2021-07-31 105521](https://user-images.githubusercontent.com/88245061/127745624-55313095-8bf0-4e68-8cd5-788822beb00e.png)

**Using Journal Roll Buttons**
Journal Entry w/ Roll Button
![Screenshot 2021-07-31 105632](https://user-images.githubusercontent.com/88245061/127745633-7e118b0f-bde0-4ca7-9392-d55d56d4e588.png)
Warning Generated by Roll Button
![Screenshot 2021-07-31 105658](https://user-images.githubusercontent.com/88245061/127745651-9d0f41e7-0b05-40e8-8227-1bba610d26b3.png)
Roll Result w/ Roll Button
![Screenshot 2021-07-31 105724](https://user-images.githubusercontent.com/88245061/127745655-ebc9f023-7601-4698-a4a5-d3342a8cc213.png)
Created Loot by Roll Button
![Screenshot 2021-07-31 105745](https://user-images.githubusercontent.com/88245061/127745665-2bfb9517-80ca-4499-a757-092caffac2aa.png)

**Using Better Tables**
Better Table w/ Text Entries
![Screenshot 2021-07-31 105832](https://user-images.githubusercontent.com/88245061/127745690-ca6a907b-95b4-413b-88bf-a05ea2d256c4.png)
Warning Generated by Roll+ Use on Better Table
![Screenshot 2021-07-31 105849](https://user-images.githubusercontent.com/88245061/127745706-3f862f67-d13a-4855-a924-e15bc0832dd5.png)
Roll Result from Better Table
![Screenshot 2021-07-31 105900](https://user-images.githubusercontent.com/88245061/127745714-f46db688-e5d9-4659-bf02-fdba52d6475e.png)
Created Loot by Better Table
![Screenshot 2021-07-31 105916](https://user-images.githubusercontent.com/88245061/127745723-f893f9bd-b65a-46ba-ae5a-837186b4a687.png)

**Comment by Daimakaicho** [comment]
Thanks for your feedback! I'll look into it next week. I think it's caused by one of the latest fix. I have an idea about how to fix it.

**Comment by Daimakaicho** [comment]
Please update to v1.8.1 and try again. Everything should be fixed.

**Comment by profdresser** [comment]
Yep! Looks like it 99.5% works now, thanks! It no longer makes loot out of text entries. _However_, it does still issue a warning when the text has a roll command:
![Screenshot 2021-08-03 180711](https://user-images.githubusercontent.com/88245061/128098404-3131629c-d094-4794-bc01-e8889b5908e8.png)
(Also weird that it calls it a table and looks in a compendium when it's a text item..._

One "fun" thing I found is that there's one, single aberration. Nothing else on the roll table does this, but I have one, single entry that, when rolled with the journal button still comes up as an item:
![Screenshot 2021-08-03 180628](https://user-images.githubusercontent.com/88245061/128098518-88c6c8fb-64f2-4d96-acd4-e6e831af6118.png)
![Screenshot 2021-08-03 180647](https://user-images.githubusercontent.com/88245061/128098525-3c142733-56e4-42c4-8ad6-d14905e6ccd5.png)

But if I roll on the table with my _macro_, then it comes up fine:
![Screenshot 2021-08-03 180609](https://user-images.githubusercontent.com/88245061/128098604-bfe71b10-ecc6-473e-a03d-dcbe98ca7870.png)

Oh, and here's an example of something else rolled off of the same table with the journal button:
![Screenshot 2021-08-03 180851](https://user-images.githubusercontent.com/88245061/128098694-b74210d2-3d2c-4eb3-b4f3-311956da47fc.png)
And the table for clarity:
![Screenshot 2021-08-03 180933](https://user-images.githubusercontent.com/88245061/128098717-97075b4f-4cf3-451d-9577-01ca09c96029.png)
Must be something in the way it's written? Not a big deal for me, just an interesting curiosity. Oh, and here's what the console logs when I roll that result with the journal button:
![Screenshot 2021-08-03 181104](https://user-images.githubusercontent.com/88245061/128098816-a4181056-417b-4373-8835-d34ab1330136.png)
